PrestaShop/PrestaShop
# Add category products number in BO list

| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop 
| Description?      | It would be interesting to have the number of products in a category. For example, this can be an indicator to know whether or not we can delete a category.
| Type?             | improvement 
| Category?         | BO 
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes https://github.com/PrestaShop/PrestaShop/issues/23514
| How to test?      | See https://github.com/PrestaShop/PrestaShop/issues/23514
| Possible impacts? | N/A


## QA team notes 

- On a shop with a lot of products, will this slow significantly the Category listing?
- We need to make sure this extra column does not stretch too much the listing (for small screens for example). Adding columns can sometime make the grid too large, and we need to scroll horizontally

By @matks 





---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/prestashop/prestashop/23515)
